akka-docs/src/main/paradox/persistence-plugins.md

```diff
@@ -0,0 +1,193 @@
+# Persistence Plugins 
+
+Storage backends for journals and snapshot stores are pluggable in the Akka persistence extension.
+
+A directory of persistence journal and snapshot store plugins is available at the Akka Community Projects page, see [Community plugins](http://akka.io/community/)
+
+Two popular plugins are:
+
+* [akka-persistence-cassandra](https://doc.akka.io/docs/akka-persistence-cassandra/current/)
+* [akka-persistence-jdbc](https://github.com/dnvriend/akka-persistence-jdbc)
+
+Plugins can be selected either by "default" for all persistent actors,
+or "individually", when a persistent actor defines its own set of plugins.
+
+When a persistent actor does NOT override the `journalPluginId` and `snapshotPluginId` methods,
+the persistence extension will use the "default" journal and snapshot-store plugins configured in `reference.conf`:
+
+```
+akka.persistence.journal.plugin = ""
+akka.persistence.snapshot-store.plugin = ""
+```
+
+However, these entries are provided as empty "", and require explicit user configuration via override in the user `application.conf`.
+For an example of a journal plugin which writes messages to LevelDB see [Local LevelDB journal](#local-leveldb-journal).
+For an example of a snapshot store plugin which writes snapshots as individual files to the local filesystem see [Local snapshot store](#local-snapshot-store).
+
```

Comment:
Do we want to bullet point these two? Just a thought.

Reply:
changed to that